packages/dockest/src/utils/selectPortMapping.ts

```diff
@@ -0,0 +1,7 @@
+import { DockerComposePortFormat } from '../@types'
+
+export const selectPortMapping = (input: DockerComposePortFormat) => {
+  if (typeof input !== 'string') return input
+  const [published, target] = input.split(':')
+  return { published: parseInt(published, 10), target: parseInt(target, 10) }
+}
```

Comment:
I'd love to see some unit tests for this piece of code

Reply:
done 👌